projects/go-lib/src/lib/components/go-button/go-button.component.ts

```diff
@@ -9,24 +9,17 @@ export class GoButtonComponent {
   @Input() buttonIcon: string;
   @Input() buttonType: string = 'button';
   @Input() buttonVariant: string;
-  @Input() useLoader: boolean;
```

Comment:
we could make this a little less breaking by keeping the binding here and just updating the html to use `useLoader` instead of isProcessing